Sources/HiPMobile/ServiceAccessLayer/ServerEndpoints.cs

```diff
@@ -26,7 +26,7 @@ public class ServerEndpoints
         /// <summary>
         /// Urlpath for the docker container running the HiP-DataStore instance
         /// </summary>
-        public const string DatastoreApiPath = "https://docker-hip.cs.uni-paderborn.de/public/datastore/api";
+        public const string DatastoreApiPath = "https://docker-hip.cs.uni-paderborn.de/develop/datastore/api";
 
```

Comment:
Why is this change necessary? Looks important enough it would deserve its own PR, since until now we've developed against the public API that doesn't change as frequently as the develop API.

Reply:
My bad this should be public, For testing, I was using develop. (Although this can only be tested with develop datastore)